books/books/middlewares.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+class RandomPayloadMiddleware:
```

Comment:
Is it possible to have a single place for this code and share it between two spiders?

Reply:
books and broad are two different scrapy projects/python packages. I could create a folder `common` with the shared middleware and symlink it?

Reply:
Yes, having a common package would be great, and since it's a package, no need to simlink - an import would work (and then maybe a more unique name would be nice, like "scrapy_bench" or "scrapy_bench_common"). It may make sense to move other top-level python files there as well, maybe as part of a separate PR.